packages/mds-stream-processor/labelers/geography-labeler.ts

```diff
@@ -14,17 +14,30 @@
     limitations under the License.
  */
 
+import db from '@mds-core/mds-db'
 import { UUID, Nullable, Telemetry } from '@mds-core/mds-types'
+import { pointInShape } from '@mds-core/mds-utils'
 import { MessageLabeler } from './types'
 
 export interface GeographyLabel {
```

Comment:
What if a geography is changed at some point? I know the idea of versioning definitions has been brought up before, not sure where we landed with it though?

Reply:
We make a new one and the new one references the previous one in the attribute `prev_geographies`.

Reply:
We should probably make it so it only annotates for published geographies, I suppose. Those are immutable.

Reply:
Pushed a change to only query for published geographies.

Reply:
Might wanna update your test to reflect it only does published geos then.